biom/tests/test_table.py

```diff
@@ -1493,6 +1493,27 @@ def test_to_dataframe_dense(self):
         obs = example_table.to_dataframe(dense=True)
         pdt.assert_frame_equal(obs, exp)
 
+    def test_to_anndata_dense(self):
+        exp = pd.DataFrame(np.array([[0.0, 1.0, 2.0], [3.0, 4.0, 5.0]]),
```

Comment:
I _think_ this is equivalent to `exp = example_table.to_dataframe(dense=True)`

Reply:
done